backends/bmv2/README.md

```diff
@@ -17,3 +17,30 @@ To run and test this back-end you need some additional tools:
 - the Python scapy library for manipulating network packets `sudo pip install scapy`
 
 - the Python ipaddr library `sudo pip install ipaddr`
+
+# Unsupported P4_16 language features
+
+Here are some unsupported features we are aware of. We will update this list as
+more features get supported in the bmv2 compiler backend and as we discover more
+issues.
+
+- nested structs:
+```
+struct s0_t {
+  bit<8> f1;
+  bit<8> f2;
+};
+
+struct s1_t {
+  s1_t s01;
+  s1_t s02;
+};
+```
+
```

Comment:
@hanw This is related to https://github.com/p4lang/p4c/issues/562
You may be right, I think this only happens for structs used in block constructor parameters. I can update my PR to be more precise. Any idea why this case is handled differently?